src/command/init/helpers.rs

```diff
@@ -8,8 +9,8 @@ pub fn display_welcome_message() {
 pub fn display_project_created_message(
     project_name: &str,
     artifacts: &Vec<Utf8PathBuf>,
-    graph_id: &str,
-    api_key: &str,
+    graph_id: &GraphId,
```

Comment:
The `api_key` has not yet been implemented, so it is prefixed with an underscore 👍